common/tier4_autoware_utils/include/tier4_autoware_utils/geometry/geometry.hpp

```diff
@@ -731,6 +735,70 @@ geometry_msgs::msg::Pose calcInterpolatedPose(
 
   return output_pose;
 }
+
+inline geometry_msgs::msg::Vector3 createVector3(const double x, double y, double z)
+{
+  return geometry_msgs::build<geometry_msgs::msg::Vector3>().x(x).y(y).z(z);
+}
+
+inline geometry_msgs::msg::Twist getTwist(
+  const geometry_msgs::msg::Vector3 & velocity, geometry_msgs::msg::Vector3 & angular)
+{
+  geometry_msgs::msg::Twist twist;
+  twist.linear = velocity;
+  twist.angular = angular;
+  return twist;
+}
+
+inline geometry_msgs::msg::TwistWithCovariance getTwistWithCovariance(
```

Comment:
Where is this function used?
I'm not sure why empty covariance is okay.

Reply:
Now related implementation is here
https://github.com/autowarefoundation/autoware.universe/blob/main/perception/radar_fusion_to_detected_object/src/radar_fusion_to_detected_object.cpp#L281
https://github.com/autowarefoundation/autoware.universe/blob/main/perception/radar_fusion_to_detected_object/src/radar_fusion_to_detected_object.cpp#L322

> I'm not sure why empty covariance is okay.
Now deteciton module in autoware do not operate covariance in DetectedObject like <https://github.com/autowarefoundation/autoware.universe/blob/main/perception/lidar_centerpoint/lib/ros_utils.cpp#L69>

Reply:
Thanks.

In that case, how about combining this line? `createTwistWithCovariance(sum_vec, twist_covariance)`
https://github.com/autowarefoundation/autoware.universe/blob/4fcc6daeed6d5485f58479129b4898c5b2904d7a/perception/radar_fusion_to_detected_object/src/radar_fusion_to_detected_object.cpp#L282

Reply:
I understand this function don't need, so I fixed at [113d56d](https://github.com/autowarefoundation/autoware.universe/pull/2049/commits/113d56db7d389a32de9a396ff81c045ccc9bcb84)